csunplugged/static/js/plugging-it-in-walkthrough.js

```diff
@@ -29,12 +29,12 @@ $(document).ready(function () {
             },
             {
                 element: "#test-case-table",
-                intro: "These are the test cases that will be run against your program.",
+                intro: "These are the test cases that will be used to see if your program is working correctly.",
                 step: 5
             },
             {
                 element: "#test-case-table tbody tr:nth-child(1)",
-                intro: "Here is the first test case.",
+                intro: "This box shows what happened for the first test case.",
```

Comment:
This is in past tense, unlike every other box. But with a change to 'happens' it doesn't quite make sense. I'm a fan of leaving it to what it was